package.json

```diff
@@ -27,7 +27,7 @@
     "ember-ajax": "^2.0.1",
     "ember-autosave": "2.0.1",
     "ember-bootstrap": "0.7.2",
-    "ember-cli": "^2.7.0",
+    "ember-cli": "^2.10.0",
```

Comment:
ember-cli and ember are versioned separately. We're still using ember (core) 2.7 in bower.json. Once we get to a working state with the current set of node modules and bower components, then I plan on doing another PR to update them to the latest LTS version and fix any issues that arise as a result of those updates.